generators/app/templates/src/app/screens/Auth/screens/Login/index.tsx

```diff
@@ -15,10 +15,11 @@ import { validationsWrapper, validateRequired, validateEmail } from '@utils/vali
 
 import './i18n';
 import styles from './styles';
+import Selectors from './selectors';
 
 function Login({ navigation }: Navigation) {
   const dispatch = useDispatch();
-  const hasLoginError = useSelector<State, boolean>(store => !!store.auth.currentUserError);
+  const hasLoginError = useSelector<State, boolean>(Selectors.hasError);
```

Comment:
What is the point of this change? Like this, I think it's the same

Reply:
to remove them from the file and ignore them, otherwise I have to access and test that method

Reply:
or leave it like that and sacrifice some coverage